api-report/debugger.api.md

```diff
@@ -128,7 +127,7 @@ export class DebugReplayController extends ReplayController implements IDebugger
 export namespace FluidDebugger {
     export function createFromService(documentService: IDocumentService): Promise<IDocumentService>;
     // (undocumented)
-    export function createFromServiceFactory(documentServiceFactory: IDocumentServiceFactory): Promise<IDocumentServiceFactory | ReplayDocumentServiceFactory>;
+    export function createFromServiceFactory(documentServiceFactory: IDocumentServiceFactory): Promise<IDocumentServiceFactory>;
```

Comment:
Not sure why this changed originally, but it is better as just `Promise<IDocumentServiceFactory>`.  I've made the return type explicit to make sure it stays this way.